commit 587127018b3ed1ce39b48aee39ee9d6b4689e3b8 (refs/changes/95/4495/1)
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2018-07-09T18:56:25-04:00 (1 year, 3 months ago)
    
    joyent/node-mooremachine#17 package.json repository does not point to Joyent

diff --git a/package.json b/package.json
index a3b1ac2..379f99a 100644
--- a/package.json
+++ b/package.json
@@ -8,7 +8,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/arekinath/node-mooremachine.git"
+    "url": "git+https://github.com/joyent/node-mooremachine.git"
   },
   "author": "Joyent, Inc",
   "contributors": [
